schema/schema.go

```diff
@@ -50,6 +51,16 @@ type Column struct {
 	Ignored Ignored
 }
 
+// ForeignKey respresents a foreign key.
+type ForeignKey struct {
+	Columns      []string
+	Name         string
```

Comment:
Suggest switching order of fields: I think Name should be first.
Also, makes it consistent with Spanner ddl.

Reply:
Done